doc: introduction: distinguishing NCS features (#12219)
nrfconnect/sdk-nrf: doc/nrf/introduction.rst

The |NCS| is a scalable and unified software development kit for building low-power wireless applications based on the Nordic Semiconductor nRF52, nRF53, nRF70, and nRF91 Series wireless devices.
It supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.

The |NCS| has the following distinguishing features:

**Advanced middleware and security**

>    The |NCS| includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).

Reviewer: In addition to middleware provided by Zephyr, adds extra libraries such as [list only these that are not available upstream].

Author: Added some, but would appreciate a more thorough look at this.

**Based on Zephyr**

>    The |NCS| integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications and samples.

Reviewer: Put as first.

**Extensible framework**

>    The |NCS| offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.

Reviewer: I'd remove this.

**Pre-certified libraries**

>    The |NCS| is optimized for pre-certified libraries, including SoftDevice, Matter, and Thread.

Reviewer: Provided pre-certified and optimized libraries.

**Robust connectivity support**

>    The |NCS| supports a wide range of connectivity technologies, such as Ant, Bluetooth® Low Energy, Bluetooth mesh, LTE-M/NB-IoT/GPS, Matter, Sidewalk, TCP/IP, Thread, and Wi-Fi®.

Reviewer: "In addition to connectivity technologies provided by Zephyr [list] it supports [list]".

**Third-party integrations**

>    The |NCS| allows for using third-party and Nordic products within the SDK, such as AWS, nRFCloud, memfault, and more.

Reviewer: ```suggestion
   The |NCS| provides integrations with third-party and Nordic products within the SDK, such as AWS, nRFCloud, memfault, and more.
```
